### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Supaboss** [barooahn/job-scraper-extension#1]
Never shows the view job after getting the details.

**Comment by barooahn** [comment]
> Never shows the view job after getting the details.

Yes, sorry still very much an Alpha build at the moment.  I'm working on the app it is supposed to support.  After finishing the app I'll move back on to this.
